ggshield/cmd/common_options.py

```diff
@@ -151,6 +152,24 @@ def log_file_callback(
 )
 
 
+minimum_severity_option = click.option(
+    "--minimum-severity",
+    "minimum_severity",
+    type=click.Choice(("LOW", "MEDIUM", "HIGH", "CRITICAL")),
+    help="Minimum severity of the policies.",
+)
```

Comment:
Shouldn't we have a default value here ? What happens if it is not set ?

Reply:
the default is set in SCAConfig -> "LOW"